app/services/create_org_and_admin_service.rb

```diff
@@ -0,0 +1,102 @@
+# class to create a new organization, user, staff account with role admin, and send email via the console
+class CreateOrgAndAdminService
```

Comment:
Hey @kasugaijin, I've got a few ideas that I want to first say are suggestions and ideas for how to improve this. I'am a big believer in coming back to improve rather than put PRs in a holding pattern. With that said, here are some considerations and ideas to counter-act them.

Consider the situation that maybe 1 of the 4 actions failed. You'll need to re-run that 1 command until it works if you realize it didn't work. This can be problematic because the data for the organization will be in a "invalid" state with missing associations or records. One way to prevent this is issue is wrapping the entire process inside of a transaction block (https://api.rubyonrails.org/classes/ActiveRecord/Transactions/ClassMethods.html). The transaction block lets you insert and update in seperate lines but rollbacks if there is an issue. I would use this transaction and use a signal method that takes a hash of information to be able to run.

Let me know if that makes sense